crates/ruff_cli/src/diagnostics.rs

```diff
@@ -64,7 +64,7 @@ pub(crate) struct Diagnostics {
     pub(crate) messages: Vec<Message>,
     pub(crate) fixed: FxHashMap<String, FixTable>,
     pub(crate) imports: ImportMap,
-    pub(crate) source_kind: FxHashMap<String, SourceKind>,
+    pub(crate) notebooks: FxHashMap<String, Notebook>,
```

Comment:
This looks reasonable. So this allows us to keep the string content on `SourceKind::Python`, since we no longer store the Python source kind on here anyway.